src/structures/GuildEvent.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const Base = require('./Base');
+const GuildEventEntityMetadata = require('./GuildEventEntityMetadata');
+const { PrivacyLevels, GuildEventEntityTypes, GuildEventStatuses } = require('../util/Constants');
+
+/**
+ * Represents an event in a {@link Guild}.
+ * @extends {Base}
+ */
+class GuildEvent extends Base {
+  /**
+   *
+   * @param {Client} client The instantiating client
+   * @param {*} data The data for the guild event
+   */
+  constructor(client, data) {
+    super(client);
+
+    /**
+     * The guild event's id
+     * @type {Snowflake}
+     */
+    this.id = data.id;
+
+    /**
+     * The id of the guild this event belongs to
+     * @type {Snowflake}
+     */
+    this.guildId = data.guild_id;
+
+    this._patch(data);
+  }
+
+  _patch(data) {
+    if ('channel_id' in data) {
+      /**
+       * The id of the stage channel this event belongs to
```

Comment:
Oh forgot this but this is why the CI is failing
```suggestion
       * The id of the stage channel this event belongs to
       * @type {Snowflake}
```